app/Http/Middleware/AuthenticateWithApiKey.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Chompy\Http\Middleware;
+
+use Closure;
+use Illuminate\Auth\AuthenticationException;
+
+class AuthenticateWithApiKey
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     *
+     * @throws \Illuminate\Auth\AuthenticationException
+     */
+    public function handle($request, Closure $next)
+    {
+        // Only protected POST, PUT, PATCH, and DELETE routes
```

Comment:
This conditional (to "only protected POST, PUT, PATCH, and DELETE routes") can be removed.